monitoring/grafana.go

```diff
@@ -12,8 +12,9 @@ import (
 const DEFAULT_PROMETHEUS_URL = "http://prometheus:9090"
 
 type GrafanaOptions struct {
-	PrometheusURL string
-	DashboardJSON string
+	PrometheusURL          string
+	DashboardJSON          string
+	ProviderSpecificConfig interface{}
```

Comment:
Is there any way we can constrain this type to something more restrictive than `interface{}`?